### DISCUSSION TITLE
Податки.

### DISCUSSION THREAD

**Discussion by MusienkoAnton** [leon-mbs/zstore#669] [Ideas]
Було б добре додати налаштування по податкам, наприклад ПДВ і т.д.
Декілька податків. Можливість у друкованих формах виводити податок оремою строкою.
Також обирати на складі зберігаються товари з податками чи без. в залежності від цього в формах і документах податок вираховується як включений або додається до вартості.

**Comment by leon-mbs** [comment]
прграмою  в  основному  користуюися  хто на  едному  податку
 
ПДВ  та  iнше  це  реальний  гемор - налаштуваннями тут не  обiйдешся
